mithril-aggregator/src/database/migration.rs

```diff
@@ -548,5 +548,18 @@ drop table db_version;
 alter table new_db_version rename to db_version;
             ",
         ),
+        // Migration 16
+        // Update `signed_entity` to remove `certificate_hash` of `artifact` JSON.
```

Comment:
Small fix here:
'// Update `signed_entity` table to remove `certificate_hash` and `created_at` from `artifact` JSON field.'